draft-ietf-webtrans-http3.md

```diff
@@ -76,7 +76,55 @@ server.  An HTTP/3 server is the server that terminates HTTP/3 connections; a
 WebTransport server is an application that accepts WebTransport sessions, which
 can be accessed via an HTTP/3 server.
 
-# Protocol Overview
+# Overview
+
+## WebTransport, QUIC and HTTP/3
+
+"QUIC: A UDP-Based Multiplexed and Secure Transport" {{!RFC9000}}
+defines QUIC security, stream multiplexing, reliable and ordered streams,
+stream HoL blocking avoidance, flow control, and congestion control.
+The transport layer provides these services to applications but does
+not constrain how streams are used.
+
+HTTP is an application-layer protocol, defined by "HTTP Semantics" {{!RFC9110}}.
+HTTP/3 has specific features that are not part of the HTTP semantics: QPACK
+header compression (static and dynamic) and Server Push. Of these, only
+static decompression is mandatory to support.
```

Comment:
QPACK static header field decompression

Reply:
And huffman literal decoding?  @vasilvv mentioned this was the most annoying part, wondering if we can create a profile or setting opting out of huffman support.

Reply:
These comments don't appear to have been incorporated. The text now says: "While WebTransport requires HTTP for the handshake, it doesn't require HTTP for anything else. HTTP/3 offers two features in addition to HTTP Semantics: QPACK header compression and Server Push. Neither of these can be used to the benefit
of WebTransport after the handshake succeeds."